AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -22,6 +22,8 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+travel.navitime.com##.app-presents-frame
+travel.navitime.com##.floating-button__map
```

Comment:
It looks like a functional button - @Yuki2718 please take a look.

Reply:
Yeah, this button shows the points of interests shown in the article in map so will be useful to user.